src/backend/web/auth.js

```diff
@@ -25,4 +27,53 @@ const basicAuth = (req, res, next) => {
   return next();
 };
 
-module.exports = { basicAuth, ensureBasicAuthCredentials };
+const ensureJwtKey = () => {
+  if (!JWT_SIGN_KEY) {
+    throw new Error("JWT_SIGN_KEY should be defined");
+  }
+};
+
+const firefighterToToken = firefighter => {
+  const token = jwt.sign({ email: firefighter.email }, JWT_SIGN_KEY, {
+    expiresIn: Math.floor(Date.now() / 1000) + 60 * 60
```

Comment:
This is nice, but given our public I don't know if they should expire at all. Ideally we want to give them less work as possible as some may be old ppl and have issues memorizing passwords. They just need 1 login for a lifetime, basically. It will never change

Reply:
Bad idea. You should manage this on the frontend. Add a service worker or something that refreshes the token every 30seconds. If somebody grabs a token, they could log in forever, even if you change the password or something.